### PR TITLE
Fix: Fix SVG uniform scale and enlarge text size

### PR DESCRIPTION
**Pull request by bbaa77770**
SB-279
SB-264

### CODE REVIEW

**Review by meiliang1123 — APPROVED** [review]
LGTM

File: src/app/models/SVGActionsFactory.js [hunk 1]
```diff
@@ -891,6 +891,13 @@ class SVGActionsFactory {
         this.onKeyMovingValue.y = 0;
     }
 
+    /**
```
Comment (parachvte) [inline src/app/models/SVGActionsFactory.js:894]: Comment without any information.

File: src/app/models/SVGActionsFactory.js [hunk 2]
```diff
@@ -958,6 +965,20 @@ class SVGActionsFactory {
         this._setSelectedElementsTransformation(t);
     }
 
+
+    /**
```
Comment (parachvte) [inline src/app/models/SVGActionsFactory.js:969]: Comment without any information.

File: src/app/models/SVGActionsFactory.js [hunk 3]
```diff
@@ -958,6 +965,20 @@ class SVGActionsFactory {
...
+     *
+     */
+
+    getSelectedElementsUniformScalingState() {
```
Comment (parachvte) [inline src/app/models/SVGActionsFactory.js:973]: Unwanted method position.

File: src/app/models/SvgModel.js [hunk 4]
```diff
@@ -317,8 +318,8 @@ class SvgModel {
             const { text, 'font-family': font, 'font-size': size } = this.relatedModel.config;
             const { scaleX, scaleY } = this.relatedModel.transformation;
             const { width, height } = this.elem.getBBox();
-            const sourceWidth = width * Math.abs(scaleX);
-            const sourceHeight = height * Math.abs(scaleY);
+            const sourceWidth = width * Math.abs(scaleX) * DEFAULT_SCALE;
```
Comment (parachvte) [inline src/app/models/SvgModel.js:321]: Add comment for `DEFAULT_SCALE`, it's a temporary solution.

**Review by parachvte — CHANGES_REQUESTED** [review]
More comment requested.

**Review by parachvte — APPROVED** [review]

**Pull request merged**